src/vaadin-progress-bar.js

```diff
@@ -0,0 +1,95 @@
+/**
+ * @license
+ * Copyright (c) 2020 Vaadin Ltd.
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+ */
+import { PolymerElement, html } from '@polymer/polymer/polymer-element.js';
+import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { ElementMixin } from '@vaadin/vaadin-element-mixin/vaadin-element-mixin.js';
+import { ProgressMixin } from './vaadin-progress-mixin.js';
+
+/**
+ * `<vaadin-progress-bar>` is a Web Component for progress bars.
+ *
+ * ```html
+ * <vaadin-progress-bar min="0" max="1" value="0.5">
+ * </vaadin-progress-bar>
+ * ```
+ *
+ * ### Styling
+ *
+ * The following shadow DOM parts are available for styling:
+ *
+ * Part name | Description
+ * ----------------|----------------
+ * `bar` | Progress-bar's background
+ * `value` | Progress-bar's foreground
+ *
+ * See [ThemableMixin – how to apply styles for shadow parts](https://github.com/vaadin/vaadin-themable-mixin/wiki)
+ *
+ * The following custom properties are available:
+ *
+ * Custom property | Description | Default
+ * ----------------|-------------|-------------
+ * `--vaadin-progress-value` | current progress value (between 0 and 1) | 0
+ *
+ * The following state attributes are available for styling:
+ *
+ * Attribute       | Description | Part name
+ * ----------------|-------------|------------
+ * `indeterminate` | Set to an indeterminate progress bar | :host
+ *
+ * @extends HTMLElement
+ * @mixes ProgressMixin
+ * @mixes ThemableMixin
+ * @mixes ElementMixin
+ */
```

Comment:
Shouldn't this also list the events (`@fires {CustomEvent} value-changed - Fired when the `value` property changes.`)?

Reply:
There's also no definitions for the event types (ProgressBarValueChanged etc)

Reply:
We do not have `notify: true` for these properties, so there is no events at all 🤷‍♂️